src/libp2p_crypto.erl

```diff
@@ -370,10 +500,218 @@ base58check_decode(B58) ->
             {error, bad_checksum}
     end.
 
+-spec pubkey_is_multisig(pubkey()) -> boolean().
+pubkey_is_multisig({multisig, _, _, _}) ->
+    true;
+pubkey_is_multisig({ecc_compact, _}) ->
+    false;
+pubkey_is_multisig({ed25519, _}) ->
+    false.
+
+%% @doc A multisig pubkey is simply an order-preserving concatanation of
```

Comment:
it's a multihash of this, along with M and N?

Reply:
Yes. Fixing.